docs/hvordan-utvikle-maler.adoc

```diff
@@ -0,0 +1,550 @@
+:docinfo: shared
+:source-highlighter: highlight.js
+:toc:
+
+= Utvikling av brevmaler i brevbakeren
+
+Brevbakeren inneholder brevmaler i form av vår egen Kotlin DSL(Domain specific language)
+DSL'en er utviklet for å kun inneholde de elementene vi trenger for å representere innholdet i brev i nav.
+
+Vi har med vilje utelatt enkelt funksjonalitet som ganging og deling ettersom vi mener at forretningslogikk bør holdes utenfor brevmaler.
+
+Mal-språket er laget spesielt for å ikke klare å lage tekniske feil i løsningen ved bestilling av ett brev, og at tekniske feil ikke fører til feil i innholdet.
+Om du definerer en god informasjons-modell for brevmalen kan du også enklere unngå funksjonelle feil.
```

Comment:
Forslag til tekst etter din setning (samme avsnitt): 

Informasjonsmodellen bør ta hensyn til virkelighetsbildet når det gjelder hvordan data er knyttet sammen og nullabillity. For eksempel om det ikke gir mening å ha verdi `A` uten verdi `B` så bør det vurderes om disse skal grupperes sammen (i en nøstet data-klasse). Om brevet _krever_ en verdi `C` så bør den vær non-nullable slik at brevproduksjonen kan feile allerede der hvor man populerer opp informasjonsmodellen.